suite-native/module-check-backup/src/components/CheckBackupTutorialStep2.tsx

```diff
@@ -27,15 +30,27 @@ type NavigationProps = StackNavigationProps<
     DeviceCheckBackupStackRoutes.CheckBackupTutorial
 >;
 
+const checkBackupUnsupportedDeviceModels: Array<DeviceModelInternal | null> = [
```

Comment:
no need for null. Am I missing something?
```suggestion
const checkBackupUnsupportedDeviceModels: Array<DeviceModelInternal> = [
```

Reply:
https://github.com/trezor/trezor-suite/pull/19963/commits/1b1a1219aca94a82c10fa5da8185455069f85627 good catch